### PR TITLE
Add a default_config command

### PR DESCRIPTION
**Pull request by ralsina**
### Pull Request Checklist

- [x] I’ve read the [guidelines for contributing](https://github.com/getnikola/nikola/blob/master/CONTRIBUTING.rst).
- [x] I updated AUTHORS.txt and CHANGES.txt (if the change is non-trivial) and documentation (if applicable).
- [x] I tested my changes.

### CODE REVIEW

File: nikola/plugins/command/default_config.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+"""Show the default configuration."""
+
+from nikola.plugin_categories import Command
+from nikola.utils import get_logger
+import nikola.plugins.command.init
+
+try:
+    print(nikola.plugins.command.init.CommandInit.create_configuration_to_string())
```
Comment (Kwpolska) [inline nikola/plugins/command/default_config.py]: I don’t think that was intended.

File: nikola/plugins/command/default_config.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+LOGGER = get_logger('default_config')
+
+class CommandShowConfig(Command):
+    """Show the default configuration."""
+
+    name = "default_config"
+
+    doc_usage = ""
+    needs_config = False
+    doc_purpose = "print the default Nikola configuration."
```
Comment (Kwpolska) [inline nikola/plugins/command/default_config.py]: (Style: uppercase `P`)

File: nikola/plugins/command/default_config.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    cmd_options = []
+
+
+    def _execute(self, options={}, args=None):
+        """Show the default configuration."""
+        try:
+            print(nikola.plugins.command.init.CommandInit.create_configuration_to_string())
+        except:
```
Comment (Kwpolska) [inline nikola/plugins/command/default_config.py]: Do `except Exception:`, it’s nicer in production code.

Comment (ralsina) [reply nikola/plugins/command/default_config.py]: Will do.

File: nikola/plugins/command/default_config.py (hunk 1 above)
Comment (ralsina) [reply nikola/plugins/command/default_config.py]: It sure was not. Silly copypasta.

**Pull request merged**